moveit_core/planning_scene/src/planning_scene.cpp

```diff
@@ -1501,10 +1501,8 @@ bool PlanningScene::processAttachedCollisionObjectMsg(const moveit_msgs::Attache
     const robot_model::LinkModel* link_model = getRobotModel()->getLinkModel(object.link_name);
     if (link_model)
     {
-      // items to build the attached object from (filled from existing world object or message)
-      std::vector<shapes::ShapeConstPtr> shapes;
-      EigenSTL::vector_Isometry3d poses;
-      moveit::core::FixedTransformsMap subframe_poses;
+      // Container for object to be created, holding shapes, their poses, and subframes
```

Comment:
What about adding "This object instance is only used for readability, and not actually attached" or a similar comment, if you think it might be confusing otherwise?

Reply:
+1 to always adding more comments